### DISCUSSION TITLE
Add the ability for users to report problems with media files

### DISCUSSION THREAD

**Discussion by phirestalker** [jellyfin/jellyfin#11056] [General]
I have NO idea if this should be here or in one of the clients, but here goes.
I run Jellyfin for my family. Occasionally they tell me that something "didn't play right." As you may imagine, that is not very useful. I would like a way for my family (from any client) to mark a media file as defective or something. Here are the features that I would like in order of simplicity.

1. Mark media with a simple checkbox or button as damaged
2. User comments
3. Client and user info
4. time and date (for log correlation)
5. better yet, attach client and server logs for the time-frame surrounding the attempted play
6. notify admin through existing system of problem report
 
I know there are third-party apps for users to request shows and movies. If there is a third-party app that will provide some or all of this functionality I would be interested to know.

**Comment by TimGels** [comment]
I want to mention that 1, 2 and 6 can be achieved already by using a third party related software called [Jellyseerr](https://github.com/Fallenbagel/jellyseerr). You might be interested in it.

**Reply by phirestalker** [reply]
Thanks, I will look into that. Just out of curiosity, why has a reporting system never been implemented in Jellyfin?

**Reply by TimGels** [reply]
It is the first time I have heard this feature request. But I would not be surprised if we were to discuss such feature, that the conclusion would be that it would be out of scope of Jellyfin.

Edit: I would suggest creating a feature request, if it doesn't exist already, over at https://features.jellyfin.org/.

**Reply by nielsvanvelzen** [reply]
Jellyfin is a home media server, if there is an issue with some media just talk to your family member downstairs.

**Reply by JPVenson** [reply]
> that the conclusion would be that it would be out of scope of Jellyfin.

Thats pretty much it. Jellyfin is a _home_ media server in the end. We really dont focus on features that model it into something else and we kinda have enough other features on our plate to serve that specific use-case. It just does not fit the model JF is build upon. And there is other 3rd party software that does that.

**Reply by phirestalker** [reply]
@nielsvanvelzen. You don't know how difficult it is to get details out of my family.

**Reply by phirestalker** [reply]
> Jellyfin is a home media server, if there is an issue with some media just talk to your family member downstairs.

> It is the first time I have heard this feature request. But I would not be surprised if we were to discuss such feature, that the conclusion would be that it would be out of scope of Jellyfin.

Is this something a plugin could be written to do, or would it need to be put into the main code? I may use third-party software to do this, I just don't think I can get them to login to something else and fill out a form or whatever.

> Edit: I would suggest creating a feature request, if it doesn't exist already, over at https://features.jellyfin.org/.
Given the responses, it seems like I would be wasting my time, but thanks.

**Reply by nielsvanvelzen** [reply]
Plugins cannot change the user interface (some do but that only works on the web client and we do not support that).

**Comment by zaggynl** [comment]
Workaround: Use the Reviews feature from this community plugin https://github.com/n00bcodr/Jellyfin-Enhanced + a custom python script which uses the API [here](https://n00bcodr.github.io/Jellyfin-Enhanced/advanced/api/?h=api#reviews-api) to  gathers the reviews.
Add a scheduled job to run and even notify/email the admin.
Downside: only works on desktop client and web client, not mobile client from my testing.
